plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -175,7 +175,8 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("impersonate").Groups(legacyGroup).Resources("serviceaccounts").RuleOrDie(),
 
 				rbac.NewRule(ReadWrite...).Groups(appsGroup).Resources("statefulsets",
-					"deployments", "deployments/scale", "deployments/rollback").RuleOrDie(),
```

Comment:
DaemonSets were not in this list before because they could evict other workloads outside of the namespace.  @liggitt on intent from sig-auth here